patientsearch/src/js/components/Search.js

```diff
@@ -140,24 +164,38 @@ async function fetchData(url) {
     }
 
     return json;
-  }
+}
 
+function Alert(props) {
+    return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+  
 export default function Search() {
     let focusInput = React.useRef(null);
     const classes = useStyles();
     const [loading, setLoading] = React.useState(false);
     const [errorMessage, setErrorMessage] = React.useState("");
+    const [popOpen, setPop] = React.useState(false);
+    const [resultOpen, setResultOpen] = React.useState(false);
     const [firstName, setFirstName] = React.useState("");
     const [lastName, setLastName] = React.useState("");
     const [dob, setDOB] = React.useState(null);
     const [success, setSuccess] = React.useState(false);
     const buttonClassname = clsx({
         [classes.buttonSuccess]: success,
     });
+    const [searchResults, setSearchResults] = React.useState([]);
 
     //TODO this is not a correct one!
-    const getLaunchURL = () => {
-        return "https://cosri-dev.cirg.washington.edu/auth/launch?launch=eyJhIjoiMSIsImIiOiI0MTcwMiIsImUiOiJTTUFSVC0xMjM0In0&iss=https%3A%2F%2Fsmart-dev-sandbox-launcher.cirg.washington.edu%2Fv%2Fr2%2Ffhir";
+    const getLaunchURL = (patientId) => {
+        //TODO ALL OF THESE, hardcoded
+        patientId = patientId || 1102;
+        let baseURL = "https://backend.cosri-demo.cirg.washington.edu";
+        let iss = "https://fhir.cosri-demo.cirg.washington.edu/hapi-fhir-jpaserver/fhir/";
+        let launchParams = "patient=" + patientId;
```

Comment:
just added some details to PT; I think the `launch` param should be calculated this way (base64 encoded JSON)
```suggestion
        let launchParam = btoa(JSON.stringify({"b":patientId}));
```

Reply:
thank you, fixed.